sql/core/src/test/resources/sql-tests/results/ansi/datetime.sql.out

```diff
@@ -662,10 +661,9 @@ struct<date_sub(DATE '2001-10-01', 7):date>
 -- !query
 select date '2001-10-01' - '7'
 -- !query schema
-struct<>
+struct<(DATE '2001-10-01' - 7):interval day>
 -- !query output
-java.time.DateTimeException
-Cannot cast 7 to DateType.
+728567 00:00:00.000000000
```

Comment:
cc @cloud-fan, this is an existing mismatch between ANSI and non-ANSI mode. For non-ANSI mode, this query will throw exception

Reply:
Let's still request at least 4 digits for year